### PR TITLE
AGW: add IRQ CPU affinity configuration

### PR DESCRIPTION
**Pull request by pshelar**
Add IRQ CPU affinity configuration for AGW.
This patch sets default configuration that restricts
datapath processing on two core. This would reduce max
packet per second performance of AGW but it improves
control plane responsiveness

Signed-off-by: Pravin B Shelar <pbshelar@fb.com>

## Test Plan
Validate on lab AGW and vagrant VM.

## Additional Information

- [ ] This change is backwards-breaking

### CODE REVIEW

File: lte/gateway/python/magma/pipelined/datapath_setup.py [hunk 1]
```diff
@@ -0,0 +1,90 @@
+"""
```
Comment (koolzz) [inline lte/gateway/python/magma/pipelined/datapath_setup.py:1]: Should this file be in python/scripts/?

File: lte/gateway/deploy/roles/magma/files/set_irq_affinity [hunk 2]
```diff
@@ -0,0 +1,265 @@
...
+		core=$i
+		set_affinity
+		((n++))
+	done
+}
+
+# now the actual useful bits of code
+
+# these next 2 lines would allow script to auto-determine interfaces
+#[ -z "$IFACES" ] && IFACES=$(ls /sys/class/net)
```
Comment (koolzz) [inline lte/gateway/deploy/roles/magma/files/set_irq_affinity:203]: rm ?

**Review by koolzz — APPROVED** [review]

File: lte/gateway/deploy/roles/magma/files/set_irq_affinity (hunk 2 above)
Comment (pshelar) [reply lte/gateway/deploy/roles/magma/files/set_irq_affinity:203]: I basically copied this script from gist, so I would like to keep it as it is for easier diff in future.

**Comment by pshelar** [comment]
the script is from: https://gist.github.com/SaveTheRbtz/8875474

**Pull request merged**